.sphinx/_templates/footer.html

```diff
@@ -101,5 +101,21 @@
 
 
     </div>
+    <div>
+      {% if github_url and github_folder and pagename and page_source_suffix %}
+          {% set contributors = get_contribs(github_url, github_folder, pagename, page_source_suffix) %}
+          {% if contributors %}
+            <a href="#" class="display-contributors">Thank you to the {{ contributors |length }} contributors!</a>
```

Comment:
Also, can we change the text to "Thanks to the ..." - I think this sounds a bit better.
Sorry for suggesting something I now don't like anymore ... :laughing:

Reply:
Not at all, I also "Thanks to the ..." sounds well.